javascript/javascript1/week2/week2-guzide/flight-booking/FligthBooking.js

```diff
@@ -0,0 +1,30 @@
+function getFullname(
+  firstname,
+  surname,
+  useFormalName = false,
+  gender = "female"
+) {
+  firstname = firstname || "";
+  surname = surname || "";
+
+  if (!firstname && !surname) {
```

Comment:
Should that be `&&` ? We know the funcion cannot wokr if we do not have both first name and surname, but can it work correctly if we have only one of those?

Reply:
Yes definitely, by using `||` problem is resolved.